src/Sylius/Bundle/UserBundle/Resources/translations/validators.en.yml

```diff
@@ -0,0 +1,35 @@
+sylius:
+    group:
+        name:
+            max: Group name must not be longer than {{ limit }} characters.
+            min: Group name must be at least {{ limit }} characters long.
+            not_blank: Please enter group name.
+
+    user:
+        email:
+            already_used: The email is already used
+            max: Email must not be longer than {{ limit }} characters.
+            min: Email must be at least {{ limit }} characters long.
+            not_blank: Please enter your email.
+            invalid: This email is invalid.
+        plainPassword:
+            not_blank: Please enter your password.
+            mismatch: The entered passwords don't match
+
+    customer:
+        first_name:
+            max: First name must not be longer than {{ limit }} characters.
+            min: First name must be at least {{ limit }} characters long.
+            not_blank: Please enter your first name.
+        last_name:
+            max: Last name must not be longer than {{ limit }} characters.
+            min: Last name must be at least {{ limit }} characters long.
+            not_blank: Please enter your last name.
+        email:
+            already_used: The email is already used
```

Comment:
This email is already (or same as above since I guess this message will only be displayed if the email belongs to a User).